src/components/analyses/listing/TableView.js

```diff
@@ -93,22 +153,29 @@ function AnalysisName(props) {
                     <IconButton
                         size="small"
                         onClick={() => handleBatchIconClick(analysis)}
-                        id={build(baseId, ids.ICONS.BATCH)}
+                        id={build(baseId, ids.ICONS.BATCH, ids.BUTTON)}
+                        className={className}
                     >
                         <UnfoldMoreIcon />
                     </IconButton>
                 </Tooltip>
-            </>
-        );
-    } else if (
-        (status === analysisStatus.SUBMITTED ||
-            status === analysisStatus.RUNNING) &&
-        interactiveUrls &&
-        interactiveUrls.length > 0
-    ) {
-        return (
-            <>
-                <NameLink />
+            )}
+            {!isDisabled && !isVICE && (
```

Comment:
If I'm reading this right, we're not allowing users to relaunch VICE analyses? Is there a reason?  I can think of a few times when something in VICE has been broken on our end and we've asked users to relaunch their analysis to see if things start working again.

Reply:
This logic decides what are the row actions available for that type of `analysis`. We certainly need relaunch for VICE. For non VICE, I have `Go to output folder, Relaunch, Drill down if Batch`. For VICE, I have `Go to output folder, Go to analysis, Extend time limit`.

Reply:
We decided to limit 3 actions per row in our UX meeting.